userspace/libsinsp/plugin.cpp

```diff
@@ -38,10 +38,21 @@ limitations under the License.
 #include "filterchecks.h"
 #include "plugin.h"
 
-#include <third-party/tinydir.h>
-
 using namespace std;
 
+// Used below--set a std::string from the provided allocated charbuf and free() the charbuf.
```

Comment:
Probably I'm missing something, but this comment suggests to me that inside this function should be a `free()` operation...

Reply:
You're right! That comment is out of date! Will fix it!